packages/traceloop-sdk/tests/datasets/test_rows_operations.py

```
@@ -0,0 +1,376 @@
+        dataset = create_mock_dataset_with_columns()
+
+        # Add some rows first
+        test_rows = get_test_rows_data()
+        dataset.add_rows_api(test_rows)
+
+        # Get the row to update
+        row_to_update = dataset.rows[0]
+        new_values = {"cmdr3ce1s0003hmp0vqons5ey": "Updated Name"}
+
+        # Try to update a row
+        with pytest.raises(Exception):
```

Ruff (0.12.2)

Do not assert blind exception: `Exception`

(B017)